example/ionic-angular-v5/package.json

```diff
@@ -14,9 +14,8 @@
     "@ionic-native/splash-screen": "^5.0.0",
     "@ionic-native/status-bar": "^5.0.0",
     "@ionic/angular": "^5.0.0",
-    "@sentry/angular-ivy": "7.81.1",
     "@sentry/capacitor": "file:.yalc/@sentry/capacitor",
-    "@sentry/cli": "^2.21.3",
```

Comment:
Was the cli removal intentional?

Reply:
here yes, it's was also on the peerDependencies from the project, not sure if wizard didn't check sentry/cli was already included on the peerDependency and just added it on the dependencies

Reply:
I'm not sure if I follow, this is the angular sample and it's the only one that doesn't have the cli now